Artesian/Artesian.SDK/Service/Query/QueryParamaters/VersionedQueryParamaters.cs

```diff
@@ -49,7 +51,9 @@ public VersionedQueryParamaters(
             LocalDateTime? versionLimit,
             FillerKindType fillerK,
             FillerConfig fillerConfig,
-            LocalDate? analysisDate
+            LocalDate? analysisDate,
+            string unitOfMeasure = null,
```

Comment:
why these added parameters have defaults while all the past one doesn't?
what makes these different?